File: chatgpt/src/main/java/com/chatgpt/app/activity/Cfg9.java
package com.chatgpt.app.activity;

import com.walhalla.landing.config.Cfg;

public class Cfg9 {

   public static Cfg cfgBetApp = new Cfg.Builder()
            .setToolbarEnabled(true)
            .setCounterTimeMs(2400)
            .setSplashScreenEnabled(false)
            .setCheckConnection(false)
            .setSwipeEnabled(false)
            .setProgressEnabled(true)
            .build();
}

File: tirauto/src/main/java/com/tirauto/transport/app/Cfg9.kt
package com.tirauto.transport.app

import com.walhalla.landing.config.Cfg


object Cfg9 {
    @JvmField
    var cfg: Cfg = Cfg.Builder()
        .setToolbarEnabled(true)
        .setCounterTimeMs(2200) //2.2s
        .setSplashScreenEnabled(
            true
        )
        .setCheckConnection(true)
        .setSwipeEnabled(false)
        .setProgressEnabled(true)
        .build()

    // https://rabota.biletiz.online/lk
    @JvmField
    var v0: IntArray = intArrayOf(
        //
    )
}
